client/src/document/index.tsx

```diff
@@ -19,7 +19,7 @@ import { BrowserCompatibilityTable } from "./ingredients/browser-compatibility-t
 // Misc
 // Sub-components
 import { Breadcrumbs } from "../ui/molecules/breadcrumbs";
-import LanguageMenu from "../header/language-menu";
+import LanguageMenu from "../ui/molecules/language-menu";
```

Comment:
I decided to have this inside `ui` because we might want to reuse it in the footer should we choose to add the language selector back there.